### ISSUE TITLE
Failing test lsp-completion-python.test.vim

### ISSUE THREAD

**Issue by kaihowl** [kaihowl/dotfiles#767]
Fails locally:

```
Test successful for test.cppRunning lsp-completion-python.test.vim
Setting up handler
Done setting up handler
Running on test.py
lsp_init: 0
About to feed keysFed keys, waitingError detected while processing command line..function RunTest[2]..Test[1]..FullTest[1]..ProtoTest[37]..Timeout:
line    1:
Test timed out%
```

**Comment by kaihowl** [comment]
Log shows:

```
Failed to load hook pylsp_completions: Python version 3.11 is currently not supported.
```

**Comment by kaihowl** [comment]
Caused by stale python 3.11 in dotfiles virtualenv on local machine. Courtesy of #703 and related workarounds.

Fixed by reinstalling virtualenv with python 3.12 version.
